core/gui/src/app/workspace/service/workflow-result-export/workflow-result-export.service.ts

```diff
@@ -20,8 +20,8 @@ import { Buffer } from "buffer";
 })
 export class WorkflowResultExportService {
   hasResultToExportOnHighlightedOperators: boolean = false;
-  hasResultToExportOnAllOperators: boolean = false;
```

Comment:
Please revert this unnecessary change!

Reply:
Sorry, it has to be used, I added to context-menu